tests/phpunit/Unit/Prompty/PromptyWalkFlowTest.php

```
@@ -220,19 +212,18 @@ public function testFlowWalkNoVisibleChildren(): void {
     $this->assertArrayNotHasKey('child', $results);
   }
 
-  #[DataProvider('dataProviderFlowWalkNumbering')]
-  public function testFlowWalkNumbering(string $number_prefix, int $step_index, string $expected_number): void {
+  #[DataProvider('dataProviderWalkFlowNumbering')]
+  public function testWalkFlowNumbering(string $number_prefix, int $step_index, string $expected_number): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$expected_number'. (undefined)

(UnusedFormalParameter)